src/groups/mqb/mqbs/mqbs_inmemorystorage.h

```diff
@@ -425,6 +421,9 @@ class InMemoryStorage BSLS_KEYWORD_FINAL : public ReplicatedStorage {
     // ACCESSORS
     //   (virtual mqbi::Storage)
 
+    /// Return the queue this storage is associated with.
+    virtual mqbi::Queue* queue() const BSLS_KEYWORD_OVERRIDE;
```

Comment:
Can the queue be null? If so, please document.

Reply:
> Can the queue be null? If so, please document.

Always been a case.  Storage exists without a queue before `setQueue` call.